purchases/src/main/kotlin/com/revenuecat/purchases/amazon/purchasing/ProxyAmazonBillingDelegate.kt

```diff
@@ -19,9 +21,10 @@ internal class ProxyAmazonBillingDelegate {
     internal var broadcastReceiver: ProxyAmazonBillingActivityBroadcastReceiver? = null
     private val filter = ProxyAmazonBillingActivityBroadcastReceiver.newPurchaseFinishedIntentFilter()
 
+    @SuppressLint("WrongConstant")
```

Comment:
This was very weird... The linter was complaining that we were passing the wrong type when passing `ContextCompat.RECEIVER_EXPORTED` even though it's the expected type...